orchestra/models/core/mixins.py

```diff
@@ -61,6 +61,9 @@ def __str__(self):
             self.phone
         )
 
+    def formatted_slack_username(self):
+        return '<@{}|{}>'.format(self.slack_user_id, self.slack_username)
+
```

Comment:
Deprecated (see https://api.slack.com/changelog/2017-09-the-one-about-usernames), they want you to just pass the user_id and let them figure out the display name.

Reply:
Updated. Thanks! I didn't know that